### PR TITLE
moved external ontology to ftx

### PR DESCRIPTION
**Pull request by feipenghe**
This PR fixes https://github.com/asyml/forte/issues/520 . 

### Description of changes
* made some changes on `ft.onto.race_multi_choice_qa_ontology` and `ft.onto.ag_news`
* reconstruct `ft` and `ftx` correspondingly. 

### Possible influences of this PR.
None.

### Test Conducted
flake8, pylint, pytest and coverage, mypy

### CODE REVIEW

File: forte/ontology_specs/ag_news.json [hunk 1]
```diff
@@ -3,9 +3,12 @@
     "imports": [
       "base_ontology.json"
     ],
+    "additional_prefixes": [
+    "ftx.onto.ag_news"
```
Comment (hunterhector) [inline forte/ontology_specs/ag_news.json:7]:
```suggestion
        "ftx"
```

File: forte/ontology_specs/base_ontology.json [hunk 2]
```diff
@@ -379,6 +379,26 @@
       "entry_name": "ft.onto.base_ontology.Title",
       "parent_entry": "forte.data.ontology.top.Annotation",
       "description": "A span based annotation `Title`, normally used to represent a title."
+    },
+        {
```
Comment (hunterhector) [inline forte/ontology_specs/base_ontology.json:383]: Let's keep the indent consistent.

File: forte/ontology_specs/race_qa.json [hunk 3]
```diff
@@ -3,14 +3,14 @@
   "imports": [
```
Comment (hunterhector) [inline forte/ontology_specs/race_qa.json:3]: Are everything in this file included in `base_ontology`? Then let's delete this file.

**Review by hunterhector — COMMENTED** [review]
A few small comments, and let's try to keep the indent consistent in the JSON files.

**Comment by hunterhector** [comment]
why did this move happens:  ...te/data/ontology/test_outputs/__init__.py → __init__.py

File: forte/ontology_specs/race_qa.json [hunk 4]
```diff
@@ -19,17 +19,17 @@
       ]
     },
     {
-      "entry_name": "ft.onto.race_multi_choice_qa_ontology.Option",
+      "entry_name": "ft.onto.base_ontology.MCOption",
       "parent_entry": "forte.data.ontology.top.Annotation"
     },
     {
-      "entry_name": "ft.onto.race_multi_choice_qa_ontology.Question",
+      "entry_name": "ft.onto.base_ontology.MCQuestion",
```
Comment (hunterhector) [inline forte/ontology_specs/race_qa.json:26]: Don't write these two times. Remove one of them

**Comment by hunterhector** [comment]
I think you haven't updated the test with the following steps:

- Remove tests/forte/data/ontology/test_outputs
- Run https://github.com/asyml/forte/blob/master/scripts/rebuild_ontology_tests.sh to rebuild tests
- Run this test to see if everything is fine. https://github.com/asyml/forte/blob/master/tests/forte/data/ontology/ontology_code_generator_test.py

**Comment by hunterhector** [comment]
besides the `__init__` files that are generated, you can delete the `.generated` files too, we don't really need them in the test case folder. Better, you could even add something in the `rebuild_ontology_tests.sh` script to automatically do that.

File: forte/ontology_specs/race_qa.json [hunk 5]
```diff
@@ -3,40 +3,20 @@
   "imports": [
     "base_ontology.json"
   ],
+    "additional_prefixes": [
```
Comment (mylibrar) [inline forte/ontology_specs/race_qa.json:6]:
```suggestion
  "additional_prefixes": [
```

**Review by mylibrar — APPROVED** [review]

**Pull request merged**